amundsen_application/static/js/components/Feedback/index.tsx

```diff
@@ -17,6 +17,8 @@ import {
 
 // TODO: Use css-modules instead of 'import'
 import './styles.scss';
+import { sendNotification } from 'ducks/notification/api/v0';
```

Comment:
I see this is for testing, let's make sure to add a `TODO: Delete after xyz` where `xyz` is whatever task allows you test this out some other way, I assume it'll be when we have the first iteration of thee "Requesting" component. Same for the comment chunk added in `toggle`.